cotainr/tests/container/test_singularity_sandbox.py

```diff
@@ -44,7 +45,9 @@ def test_setup_log_dispatcher(self):
 
 class TestContext:
     def test_add_verbosity_arg(self, capsys, patch_disable_stream_subprocess):
-        with SingularitySandbox(base_image="my_base_image_6021"):
+        sandbox = SingularitySandbox(base_image="my_base_image_6021")
+        sandbox.architecture = "test"
```

Comment:
This segment being repeated so many times looks like `architecture` should be made an optional argument for the class instead?

Reply:
Then it could be a good idea to make this stanza a pytest fixture, to avoid repetition.